app/code/Magento/Contact/view/frontend/layout/contact_index_index.xml

```diff
@@ -13,6 +13,7 @@
         <referenceContainer name="content">
             <block class="Magento\Contact\Block\ContactForm" name="contactForm" template="Magento_Contact::form.phtml">
                 <container name="form.additional.info" label="Form Additional Info"/>
+                <block class="Magento\Captcha\Block\Captcha" name="captcha" after="-" /> 
```

Comment:
This block is already added in `Magento/Captcha/view/frontend/layout/contact_index_index.xml`

Reply:
Hi @orlangur ,

Author of issue has mentioned this "disable module magento_captcha" in precondition. So, default behaviour is ignore.

Can we add dependency in reCaptcha module? so, by default captcha module can't be disable. 

Please guide me for the same.

Thanks.

Reply:
@hiren0241 I see... Definitely, layout instruction must not be duplicated in some enabled module. reCaptcha module must depend on `Magento_Captcha` if it required for it to function properly.

Reply:
@orlangur, 

Once again thanks for quick response. 

Can I make separate commit  to add dependency in that module, and delete previously applied changes. So that we can go further in this PR or should I have to refer this PR to this issue: https://github.com/magento/magespecialist_ReCaptcha/issues/21 ?

Please guide me for the same.

Thanks.

Reply:
@hiren0241 please proceed with PR in https://github.com/magento/magespecialist_ReCaptcha/pulls I believe.

Reply:
Hi @orlangur,

As per your suggestion, I have created PR. Kindly do have a look at https://github.com/magento/magespecialist_ReCaptcha/pull/24 

Thanks.